bin/scrape_software_versions.py

```diff
@@ -18,6 +18,8 @@
     'StringTie': ['v_stringtie.txt', r"(\S+)"],
     'Preseq': ['v_preseq.txt', r"Version: (\S+)"],
     'RSeQC': ['v_rseqc.txt', r"read_duplication.py ([\d\.]+)"],
+    'DupRadar': ['v_dupRadar.txt', r"version (\S+)"],
```

Comment:
Would be good to keep the naming consistent i.e. `dupRadar`

Reply:
Also, `r"version (\S+)"` would be the wrong regex here because the command output doesnt contain the `version` string.

Reply:
You will probably also see that you need to add `dupRadar` and `edgeR` to two sections in `scrape_software_versions.py`. Then that should be it 👍